docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,19 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service DCP Rollback to Zero (CB90067)
+
+*Background*: A service has been forced to DCP rollback to zero.
+This may be undesirable because it means the service is attempting to resynchronize the entirety of the data set.
+
+*Condition*: A DCP rollback to zero is seen in the `memcached` logs.
+
+*Remediation*: This is typically a symptom of another problem, you should monitor your cluster closely for any further issues.
+If you experience any, then please contact Couchbase Technical Support.
+
+*Further Reading*: https://blog.couchbase.com/couchbase-dcp-rollback-qa-tests/[Couchbase DCP Rollback]
```

Comment:
new xref link needed?

Reply:
No - blog.couchbase.com isn't part of docs.couchbase.com so we can't xref to it.

Reply:
with blog posts we can't create xref links, we don't have anything in the docs about dcp rollback so this was the best option